Sources/Plasma/PubUtilLib/plNetClientComm/plNetClientComm.h

```diff
@@ -76,6 +76,11 @@ struct NetCommPlayer {
     plString    playerName;
     plString    avatarDatasetName;
     unsigned    explorer;
+
+    NetCommPlayer() { }
+    NetCommPlayer(unsigned id, const plString& name, const plString& shape, unsigned ex)
+        : playerInt(id), playerName(name), avatarDatasetName(shape), explorer(ex)
```

Comment:
I tried doing just that on something in VC++2013 awhile back and it didn't like the initializer list :cry: